src/screens/Home/Home.js

```diff
@@ -1,788 +1,86 @@
-                          listKey="sablier"
-                        />
-                      }
-                      onPress={toggleSablier}
-                      open={!hideSablier}
-                    />
-                  }
-                  {!!hasRariDeposits && !!isSmartWalletActive &&
-                    <CollapsibleSection
-                      label={t('rariContent.depositsList.title')}
-                      showLoadingSpinner={isFetchingRariData}
-                      labelRight={isFetchingRariData ? null : t('button.viewAll')}
-                      onPressLabelRight={() => { navigation.navigate(RARI_DEPOSIT); }}
-                      collapseContent={
-                        <FlatList
-                          data={rariDeposits}
-                          keyExtractor={(item) => item.pool}
-                          renderItem={this.renderRariPool}
-                          initialNumToRender={2}
-                          listKey="rari"
-                        />
-                      }
-                      onPress={toggleRari}
-                      open={!hideRari}
-                    />
-                  }
-                  {!!purchasedLiquidityPools.length && !!isSmartWalletActive &&
-                    <CollapsibleSection
-                      label={t('liquidityPoolsContent.depositsList.title')}
-                      showLoadingSpinner={isFetchingLiquidityPoolsData}
-                      labelRight={isFetchingLiquidityPoolsData ? null : t('button.viewAll')}
-                      onPressLabelRight={() => { navigation.navigate(LIQUIDITY_POOLS_SCREEN); }}
-                      collapseContent={
-                        <FlatList
-                          data={purchasedLiquidityPools}
-                          keyExtractor={(item) => item.pool.name}
-                          renderItem={this.renderLiquidityPool}
-                          initialNumToRender={2}
-                          listKey="liquidityPools"
-                        />
-                      }
-                      onPress={toggleLiquidityPools}
-                      open={!hideLiquidityPools}
-                    />
-                  }
-                </React.Fragment>
-              )}
-              flatListProps={{
-                refreshControl: (
-                  <RefreshControl
-                    refreshing={false}
-                    onRefresh={this.refreshScreenData}
-                  />
-                ),
-                onScroll,
-                scrollEventThrottle: 16,
-              }}
-            />
-          )}
-        </ContainerWithHeader>
-      </React.Fragment>
-    );
-  }
+      <Content>
+        {/* Temporary content */}
+        <MainContent>
+          {/* eslint-disable-next-line i18next/no-literal-string */}
+          <Text>Hello Pillar!</Text>
+        </MainContent>
+
+        {/* Temporary navigation section */}
+        {/* eslint-disable-next-line i18next/no-literal-string */}
```

Comment:
nit: we disable this rule plenty of times throughout the codebase, can we disable it globally?

Reply:
I'm not a fan of this rule, however it seems to be added by intent: https://github.com/pillarwallet/pillarwallet/pull/2213

@IAmKio maybe we could disable this rule it globally, wdyt?